app/_components/layout/Footer.tsx

```diff
@@ -0,0 +1,77 @@
+import NavLinks from '@/_components/ui/NavLinks';
```

Comment:
this is not valid import ?
```suggestion
rename the file of nav links ..
```